typescript/graphql-sdl-first/src/schema.ts

```diff
@@ -214,8 +214,3 @@ interface UserCreateInput {
   posts?: PostCreateInput[],
 }
 
```

Comment:
Is there any reason why we're not using `makeExecutableSchema`?

Reply:
![image](https://user-images.githubusercontent.com/27310414/131698719-690ed604-5a87-4496-bd39-0947b74a34e1.png)
![image](https://user-images.githubusercontent.com/27310414/131707014-fb79171e-d55b-421e-9730-6ccfd6b04ca0.png)
https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md

It got removed in this version